src/components/BlogCard/BlogCard.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import styles from './BlogCard.module.scss'
+import { BlogProps, RootState } from '../../models/models'
+import classNames from 'classnames'
+import { useSelector } from 'react-redux'
```

Comment:
It is always good to move the external pkg imports to the top and have ours at the bottom